Makefile

```diff
@@ -43,6 +43,11 @@ do_tests::
 	$(MAKE) -k -C tests
 do_tests::
 	$(MAKE) -k -C examples
+# increase coverage
+do_tests::
+	$(MAKE) -k -C tests/test_cases/test_cocotb/ COCOTB_LOG_LEVEL=DEBUG > test_cocotb_DEBUG.log
+do_tests::
+	$(MAKE) -k -C tests/test_cases/test_cocotb/ COCOTB_SCHEDULER_DEBUG=1 > test_cocotb_SCHEDULER_DEBUG.log
```

Comment:
It depends on how good we want the tests to be. Successfully running our testcases with DEBUG on means the switches don't break anything, but we don't check that they actually lead to debug output - this could be done with a few ``grep``s in the logfile.